docs/one-off-tasks.md

```diff
@@ -0,0 +1,22 @@
+# One-off tasks
+
+For these, we're using Flask commands, which live in [`/app/commands.py`](../app/commands.py).
+
+This includes things that might be one-time operations! Using a command allows the operation to be tested, 
+both with `pytest` and with trial runs.
+
+To run a command on cloud.gov, use this format:
+
+```
+cf run-task CLOUD-GOV-SPACE --commmand "YOUR COMMAND HERE" --name YOUR-COMMAND
```

Comment:
[issue] I think the app name needs to be included in `run-task`, instead of the space, right??